lxd/instance/drivers/driver_common.go

```diff
@@ -711,29 +712,28 @@ func (d *common) isStartableStatusCode(statusCode api.StatusCode) error {
 	return nil
 }
 
-// startupSnapshot triggers a snapshot if configured.
-func (d *common) startupSnapshot(inst instance.Instance) error {
+func (d *common) getSnapNameAndExpiry(inst instance.Instance) (string, time.Time, error) {
```

Comment:
Lets call this `getStartupSnapNameAndExpiry()` so its clear its to do with startup snapshots. Also please retain and update the comment.

Reply:
Lets also return a *time.Time so we can return nil for that var on error.